README.md

```diff
@@ -40,131 +53,177 @@ FIEDLER, Niklas, et al. [ImageTagger: An Open Source Online Platform for Collabo
-
-for additional steps on some releases see instructions
-in [UPGRADE.md](https://github.com/bit-bots/imagetagger/blob/master/UPGRADE.md)
-
-## Install
-
-Checkout the latest release:
-
-```
-git checkout v0.x
-```
-
-In our production Senty is used for error reporting (pip install sentry-sdk).
-django-auth-ldap is used for login via ldap
-uwsgi is used to serve the app to nginx
-
-Install Python Dependencies:
-
-```
-pip3 install -r requirements.txt
-```
-
-Copy settings.py.example to settings.py in the imagetagger folder:
-
-```
-cp imagetagger/settings.py.example imagetagger/settings.py
-```
-
-and customize the settings.py.
-
-The following settings should probably be changed:
-
-+ The secret key
-+ The DEBUG setting
-+ The ALLOWED\_HOSTS
-+ The database settings
-+ The UPLOAD\_FS\_GROUP to the id of the group that should access and create the uploaded images
-
-For the database, postgresql is used. Install it by running `sudo apt install postgresql` on Debian based operating systems. A default database cluster is automatically initialized.
-
-Other systems may require different commands to install the package and the database cluster may
-have to be initialized manually (e.g. using `sudo -iu postgres initdb --locale en_US.UTF-8 -D '/var/lib/postgres/data'`).
-
-To start the postgresql server, run `sudo systemctl start postgresql.service`. If the server should always be started on boot, run `sudo systemctl enable postgresql.service`.
-
-Then, create the user and the database by running
-
-`sudo -iu postgres psql`
-
-and then, in the postgres environment
-
-```
-CREATE USER imagetagger PASSWORD 'imagetagger';
-CREATE DATABASE imagetagger WITH OWNER imagetagger ENCODING UTF8;
-```
-
-where of course the password and the user should be adapted to the ones specified in the database settings in the settings.py.
-
-To initialize the database, run `./manage.py migrate`
-
-To create an administrator user, run `./manage.py createsuperuser`.
-
-`./manage.py runserver` starts the server with the configuration given in the settings.py file.
-
-To create annotation types, log into the application and click on Administration at the very bottom of the home page.
-
-For **production** systems it is necessary to run the following commands after each upgrade
-
-```bash
+## Installing and running ImageTagger
+ImageTagger can be installed and run locally (best for development), in a docker container or in Kubernetes 
+(used in our deployment).
+
+### Locally
+
+In some of the following code snippets, the `DJANGO_CONFIGURATION` environment variable is exported.
+This defines the type of deployment by selecting one of our predefined configuration presets.
+If ImageTagger is running in a development environment, no export is necessary.
+
+1.  #### Install the latest release
+    
+    You should probably do this in a [virtual environment](https://virtualenv.pypa.io/en/stable/) 
+    ```shell
+    git checkout v…
+    pip3 install -r imagetagger/requirements.txt
```

Comment:
I would probably add `cd imagetagger` here since all the commands below are also executed from the imagetagger folder.

Reply:
Now you also have to change the pip command.